test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/file_tailing_test.go

```diff
@@ -218,9 +219,34 @@ func (s *LinuxFakeintakeSuite) LogRecreateRotation() {
 	t.Logf("Permissions granted for new log file.")
 
 	// Generate new logs
-	appendLog(s, "hello-world-new-content", 1)
+	s.AppendLog("hello-world-new-content", 1)
 
 	// Check intake for new logs
-	checkLogs(s, "hello", "hello-world-new-content", true)
+	s.CheckLogs("hello", "hello-world-new-content", true)
 
 }
+
+// AppendLog calls the utility function to append a log.
+func (s *LinuxFakeintakeSuite) AppendLog(content string, recurrence int) {
+	utils.AppendLog(s, content, recurrence)
+}
+
+// CheckLogFilePresence calls the utility function to check the presence of a log file.
+func (s *LinuxFakeintakeSuite) CheckLogFilePresence(logPath string) {
+	utils.CheckLogFilePresence(s, logPath)
+}
+
+// CheckLogs calls the utility function to check the presence of a log inside the intake.
+func (s *LinuxFakeintakeSuite) CheckLogs(service, content string, expectLogs bool) {
+	utils.CheckLogs(s, service, content, expectLogs)
+}
+
+// CleanUp calls the utility function to clean up the test environment.
+func (s *LinuxFakeintakeSuite) CleanUp() {
+	utils.CleanUp(s)
+}
+
+// IsDevMode returns true if the test suite is running in dev mode.
+func (s *LinuxFakeintakeSuite) IsDevMode() bool {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
We should have this at `e2e.Suite` level